devops/helms/messagebroker/templates/deployment.yaml

```diff
@@ -50,6 +50,31 @@ spec:
                 configMapKeyRef:
                   name: {{ include "messagebroker.fullname" . }}-configmap
                   key: natsUrl
+            - name: WITH_SWAGGER
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "messagebroker.fullname" . }}-configmap
+                  key: withSwagger
+            - name: WEB_URL
```

Comment:
WEB3_URL

Reply:
So there is a mistake in Readme description also :p

Reply:
Please push the README fix too, good catch 👍